### PR TITLE
feat: add help modal

### LINKED ISSUE #68: @kind/feature - Create Lander Modal
**Is your feature request related to a user or engineer story?** : User 

**What story does this feature enable?**
As a user on my first visit I should be presented a modal that offers help

**Describe the surrounding work**

Add a modal as found in the [figma](https://www.figma.com/file/4FaeOKbyhTvMrFCjWXg0Wk/Staking-UI-Tabs-V2?node-id=519%3A17130&t=Eg1FiOibkgVHo9fl-0) and as exampled in the img below:

This modal should only load once for a user and a session cookie should prevent it's default pop up in the future

Menus, Text, And Links should be available in the figma prototype

**Additional context**

image

### PR DESCRIPTION
**Pull request by ACatThatPrograms**
Resolves #68

### CODE REVIEW

File: alca-app/src/components/LanderHelpModal.jsx [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+            setModalPage(modalPageNames.welcome);
+        }
+    }, [isModalOpen, modalPageNames.welcome]);
+
+    const ModalHeader = () => (
+        <>
+            <Typography id="modal-modal-title" variant="h5" component="h2">
+                Welcome to AliceNet
+            </Typography>
+            <Typography id="modal-modal-title" variant="p" component="h2">
```
Comment (everton0991) [inline alca-app/src/components/LanderHelpModal.jsx:51]: Duplicated ids

File: alca-app/src/components/LanderHelpModal.jsx [hunk 2]
```diff
@@ -0,0 +1,273 @@
+// import React from 'react';
```
Comment (everton0991) [inline alca-app/src/components/LanderHelpModal.jsx:1]: This can be removed

Comment (ACatThatPrograms) [reply alca-app/src/components/LanderHelpModal.jsx:1]: Fixed :100:

File: alca-app/src/components/LanderHelpModal.jsx [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        return (
+            <Box>
+                <ModalContent sx={{ mt: 2, maxWidth: "75ch" }} />
+            </Box>
+        );
+    };
+
+    const ModalActions = () => {
+        return (
+            <Box sx={{ display: "flex", mt: 2, justifyContent: "flex-end" }}>
```
Comment (adrianbona) [inline alca-app/src/components/LanderHelpModal.jsx:235]: Most SX attributes can be moved to Box's attributes

**Pull request merged**